securitycenter/notificationsv2/notifications_test.go

```diff
@@ -0,0 +1,326 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notificationsv2
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	securitycenter "cloud.google.com/go/securitycenter/apiv2"
+	"cloud.google.com/go/securitycenter/apiv2/securitycenterpb"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"github.com/google/uuid"
+)
+
+func orgID(t *testing.T) string {
+	orgID := os.Getenv("GCLOUD_ORGANIZATION")
+	if orgID == "" {
+		t.Fatal("GCLOUD_ORGANIZATION not set")
+	}
+	orgID = strings.TrimSpace(orgID)
+	return orgID
+}
+
+func projectID(t *testing.T) string {
+	projectID := os.Getenv("SCC_PUBSUB_PROJECT")
+	if projectID == "" {
+		t.Fatal("SCC_PUBSUB_PROJECT not set")
+	}
+	projectID = strings.TrimSpace(projectID)
+	return projectID
+}
+
+func pubsubTopic(t *testing.T) string {
+	pubsubTopic := os.Getenv("SCC_PUBSUB_TOPIC")
+	if pubsubTopic == "" {
+		t.Fatal("SCC_PUBSUB_TOPIC not set")
+	}
+	pubsubTopic = strings.TrimSpace(pubsubTopic)
+	return pubsubTopic
+}
+
+func pubsubSubscription(t *testing.T) string {
+	pubsubSubscription := os.Getenv("SCC_PUBSUB_SUBSCRIPTION")
+	if pubsubSubscription == "" {
+		t.Fatal("SCC_PUBSUB_SUBSCRIPTION not set")
+	}
+	pubsubSubscription = strings.TrimSpace(pubsubSubscription)
+	return pubsubSubscription
+}
+
+func createTestNotificationConfig(buf *bytes.Buffer, orgID string, pubsubTopic string, configID string) error {
```

Comment:
is this the same as the sample function `createNotificationConfig`?

I dont think this is necessary, since we can just call `createNotificationConfig` in these tests. what was your reasoning for creating this helper method?

the same is true for deleteNotificationConfig and the cleanupNotificationConfig methods.

Reply:
I thought of creating a sample notification in the test itself for testing purposes and have written the helper function. But you are correct, we can reuse the existing sample function. Removed the function and refactored the file.